spec/models/location_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+describe Location do
+  context "validations" do
+    it { is_expected.to validate_presence_of(:creator) }
+    it { is_expected.to validate_presence_of(:physical_address) }
+    it { is_expected.to validate_presence_of(:city) }
+    it { is_expected.to validate_presence_of(:state) }
+    it { is_expected.to validate_presence_of(:country) }
+    it { is_expected.to validate_presence_of(:ethereum_address) }
```

Comment:
@DavidKnott Do we need the ethereum address to be unique?  I'm having a hard time wrapping my head around the pros/cons of it being unique.  What do you think?